validphys2/src/validphys/theorydbutils.py

```diff
@@ -5,80 +5,99 @@
 low level utilities for querying the theory database file and representing the
 data as a python object.
 """
+from dataclasses import asdict, dataclass
+from functools import lru_cache
 from pathlib import Path
 
-# Keep the sqlite3 dependence in one location
-import sqlite3
-
 import pandas as pd
 
+from validphys.utils import parse_yaml_inp
+
+
+@dataclass(frozen=True)
+class _TheoryCard:
+    ID: int
+    PTO: int
+    FNS: str
+    DAMP: int
+    IC: int
+    ModEv: str
+    XIR: float
+    XIF: float
+    NfFF: int
+    MaxNfAs: int
+    MaxNfPdf: int
```

Comment:
I would start removing parameters that we already know are useless. For example this `MaxNfPdf` but also `MaxNfAs` and `EScaleVar` (and most likely others...)

Reply:
I will do that in subsequent PRs.  The idea is that this leaves validphys essentially unchanged.